clang-tools-extra/docs/ReleaseNotes.rst

```diff
@@ -224,6 +224,13 @@ New checks
   Recommends the smallest possible underlying type for an ``enum`` or ``enum``
   class based on the range of its enumerators.
 
+- New :doc:`readability-use-std-min-max
+  <clang-tidy/checks/readability/use-std-min-max>` check.
+
+  Replaces certain conditional statements with equivalent ``std::min`` or
+  ``std::max`` expressions, improving readability and promoting the use of
+  standard library functions.
```

Comment:
"improving readability and promoting the use of  standard library functions." that part is not needed here.